test/integration/services/alert.js

```diff
@@ -0,0 +1,74 @@
+describe('Alert Service', function() {
+  specify('Displaying', function() {
```

Comment:
only line missing is the delay timeout.. should be able to set the delay, hard set a wait to the delay + animation time and check `should('not.exist')`